homeassistant/components/zha/core/device.py

```diff
@@ -819,7 +819,7 @@ async def _async_group_binding_operation(
                 fmt = f"{log_msg[1]} completed: %s"
             zdo.debug(fmt, *(log_msg[2] + (outcome,)))
 
-    def log(self, level: int, msg: str, *args: Any, **kwargs: dict) -> None:
+    def log(self, level: int, msg: str, *args: Any, **kwargs: Any) -> None:
```

Comment:
For `**kwargs` type hints, we need to specify only the value type of the dict, not the dict itself.